aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs

```diff
@@ -230,6 +231,11 @@ crate::gas_schedule::macros::define_gas_parameters!(
             { 15.. => "max_total_dependency_size" },
             1024 * 1024 * 12 / 10, // 1.2 MB
         ],
+        [
+            keyless_multiplier: NumBytes,
+            { 16.. => "keyless_multiplier" },
```

Comment:
Need to document this new gas change in the comment section for `LATEST_GAS_FEATURE_VERSION`